src/dss/list.py

```diff
@@ -21,34 +24,54 @@ def list_notebooks(lightkube_client: Client, wide: bool = False) -> None:
                                Defaults to False.
     """
     try:
-        deployments = lightkube_client.list(Deployment, namespace=DSS_NAMESPACE)
+        deployments = list(
+            lightkube_client.list(
+                Deployment,
+                namespace=DSS_NAMESPACE,
+                labels={NOTEBOOK_LABEL: lightkube.operators.exists()},
+            )
+        )
     except ApiError as e:
         logger.error(f"Failed to list notebooks: {e}")
         return
 
-    notebook_list = []
+    if not deployments:
+        # Use stdout for table so the output can be piped properly
+        print("No notebooks found in dss")
```

Comment:
I don't think you need "in dss" here, since you're calling `dss list`.  If you want dss in the message, I think "No dss notebooks found" is more correct

```suggestion
        print("No notebooks found")
```